test/cloudant.test.js

```diff
@@ -295,6 +310,124 @@ describe('cloudant connector', function() {
       });
     });
   });
+
+  describe('allow numerical `id` value', function() {
+    var data = [{
+      id: 1,
+      name: 'John Chow',
+      age: 45,
+    }, {
+      id: 5,
+      name: 'Kelly Johnson',
+      age: 25,
+    }, {
+      id: 12,
+      name: 'Michael Santer',
+      age: 30,
+    }];
+
+    before(function(done) {
+      SimpleEmployee.create(data, function(err, result) {
+        if (err) done(err);
+        done(null, result);
```

Comment:
hmm, any reason why callback `result` here? `SimpleEmployee.create(data,done);` would be enough IMO.